storybookjs/storybook
# UI: Reflect search in URL

Issue: #16287

Integrate way to reflect search("filter") in URL bar in order to share the URL for hosted Storybook instances.

## What I did
- Grab the `Sidebar` `Search` `initialQuery` from `URLSearchParams` and use the `filter` value.
- Tap into `Downshift` render to update the URL bar with `window.history.replaceState`. 

## How to test


If your answer is yes to any of these, please make sure to include it in your PR.

Tests are under `Search.test.js`. Tests populating the input field in addition to update `window.location` via mocked `window.history` 👍
